src/CMakeLists.txt

```diff
@@ -14,39 +14,53 @@ add_executable(runtime
 )
 
 add_library(vgg_internal OBJECT
+  Controller/Controller.cpp
   Exec/VggExec.cpp
   Model/DocumentModel.cpp
+  Model/JsonDocument.cpp
   Model/JsonSchemaValidator.cpp
+  Model/SchemaValidJsonDocument.cpp
+  Model/SubjectJsonDocument.cpp
+  Model/VggWork.cpp
+  Sdk/VggSdk.cpp
   PlatformAdapter/Helper/StringHelper.cpp
 )
 target_include_directories(vgg_internal PUBLIC 
   ${CMAKE_CURRENT_SOURCE_DIR}/Api
+  ${CMAKE_CURRENT_SOURCE_DIR}/Controller
   ${CMAKE_CURRENT_SOURCE_DIR}/Exec
+  ${CMAKE_CURRENT_SOURCE_DIR}/Infrastructure
   ${CMAKE_CURRENT_SOURCE_DIR}/Model
+  ${CMAKE_CURRENT_SOURCE_DIR}/Sdk
```

Comment:
The above include directories are not necessary. As to the usage, I think
```
#include "Model/RawJsonDocument.hpp"
```
is more preferred than
```
#include "RawJsonDocument.hpp"
```

Reply:
Fixed.